.gitpod.yml

```diff
@@ -0,0 +1,26 @@
+tasks:
+  - name: Run Utopia Docker Stack
+    init: |
+      docker compose pull
+      docker compose build
+    command: |
+      docker run --rm --interactive --tty \
+      --volume $PWD:/app \
+      composer update \
```

Comment:
we need to use `composer install` here 👍🏻

Reply:
Fixed.